MozCamp/bootstrap.js

```diff
@@ -73,9 +73,22 @@ function startup(aData, aReason) {
 		}
 	},false);
 	
-	let date = new Date(Date.now() - (4 * 30 * 24 * 60 * 60 * 1000)).toISOString().split('T').shift().replace('-','','g');
-	xhr.open('GET', 'http://www.flickr.com/search?data=1&q=mozilla%20summit%20OR%20mozcamp&s=int&mt=photos&d=taken-'+date+'-&append=1', true);
-	xhr.send();
+	let date = new Date(Date.now() - (4 * 30 * 24 * 60 * 60 * 1000));
+	let formData = new flickerSearchForm();
+	formData.append("extras", "url_o,url_h");
+	formData.append("tags", "mozsummit,mozcamp");
```

Comment:
With free text I wasn't able to search as the old code did. So I switched to tag based search from free text based search. The new result may differ from the old result set.